src/dynarec/arm64/dynarec_arm64_avx_0f38.c

```diff
@@ -150,13 +227,14 @@ uintptr_t dynarec64_AVX_0F38(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, i
             GETED(0);
             GETVD;
             UXTBw(x1, vd);
-            CMPSw_U12(x1, rex.w?64:32);
-            CSETxw(x2, cPL);
+            CMPSw_U12(x1, rex.w?63:31);
+            CSETxw(x2, cGT);
```

Comment:
are you sure?

Reply:
I was testing `x1>=64` (or 32), and you changed it to `x1>63` (or 31). That's strictly the same thing, so I don't see why it should be changed.